src/server/service/kv.rs

```diff
@@ -2373,6 +2382,38 @@ fn needs_reject_raft_append(reject_messages_on_memory_ratio: f64) -> bool {
     false
 }
 
+struct HealthFeedbackAttacher {
+    health_controller: HealthController,
+    last_feedback_time: Option<Instant>,
+}
+
+impl HealthFeedbackAttacher {
+    fn new(health_controller: HealthController) -> Self {
+        Self {
+            health_controller,
+            last_feedback_time: None,
+        }
+    }
+
+    fn attach_if_needed(&mut self, resp: &mut BatchCommandsResponse) {
```

Comment:
Do you  mean that not every response needs to return the slow score.

Reply:
yes